services/api/internal/user/routes/UpdateUserInfo.go

```diff
@@ -0,0 +1,35 @@
+package routes
+
+import (
+	"context"
+	"net/http"
+
+	pb "github.com/CrabStash/crab-stash-protofiles/user/proto"
+	valid "github.com/asaskevich/govalidator"
+	"github.com/gin-gonic/gin"
+)
+
+func UpdateUserInfo(ctx *gin.Context, c pb.UserServiceClient) {
+	payload := pb.UpdateUserInfoRequest{}
+	uuid, _ := ctx.Get("uuid")
+	payload.UserID = uuid.(string)
+
+	if err := ctx.BindJSON(&payload); err != nil {
+		ctx.JSON(http.StatusBadRequest, gin.H{"status": http.StatusBadRequest, "response": gin.H{"error": err.Error()}})
+		return
+	}
+	_, err := valid.ValidateStruct(&payload)
+
+	if err != nil {
+		ctx.JSON(http.StatusBadRequest, gin.H{"status": http.StatusBadRequest, "response": gin.H{"error": err.Error()}})
```

Comment:
Jeżeli Payload który dostajesz nie binduje się do structa powinieneś dać raczej 400 - Bad Request, popraw to w odpowiednich miejscach

Reply:
^ zdecydowanie, ułatwi nam to debugowanie